src/main/java/mate/controller/car/DisplayCarsController.java

```diff
@@ -0,0 +1,22 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+
+public class DisplayCarsController extends HttpServlet {
```

Comment:
Better to rename to GetAllCarsController

Reply:
Sorry, but really dont see any reason

Reply:
You controller doesn't display anything it's job to get all cars from db. It is the jsp page that display cars

Reply:
> Your controller doesn't display anything it's job to get all cars from db. It is the jsp page that display cars

\+